src/components/SettingsDialog/Appearance/Appearance.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from "classnames";
+import {t} from "i18next";
+import {useState} from "react";
+import {SettingsButton} from "../Components/SettingsButton";
+import {SettingsDropdown} from "../Components/SettingsDropdown";
+import {SettingsToggle} from "../Components/SettingsToggle";
+import "../SettingsDialog.scss";
+import "./Appearance.scss";
+
+export const Appearance = () => {
+  const [syncMode, setSyncMode] = useState<boolean>(false); // placeholder
+  const [showNotifications, setshowNotifications] = useState<boolean>(false); // placeholder
+  const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+  return (
+    <div className={classNames("settings-dialog__container", "accent-color__lean-lilac")}>
+      <header className="settings-dialog__header">
+        <h2 className="settings-dialog__header-text">{t("SettingsDialog.Appearance")}</h2>
+      </header>
+
+      <div className="appearance-container">
+        <SettingsButton
+          className="appearance-settings_sync-button"
+          onClick={() => {
+            setSyncMode(!syncMode);
```

Comment:
It's more safe to do `setSyncMode(prevVal => !prevVal)`

Reply:
Also, this doesn't seem to do anything. When will the functionality be implemented?